contracts/mocks/ERC20SnapshotMock.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.24;
+
+import "../token/ERC20/ERC20Snapshot.sol";
+
+
+contract ERC20SnapshotMock is ERC20Snapshot {
+
+  constructor(address initialAccount, uint256 initialBalance) public {
+    _mint(initialAccount, initialBalance);
+  }
+
+  function mint(address account, uint256 amount) public {
+    _mint(account, amount);
+  }
+
+  function burn(address account, uint256 amount) public {
+    _burn(account, amount);
+  }
+}
```

Comment:
Does not include `_burnFrom` from the current v2 RC2 OZ

Reply:
It's a mock for unit test purpose so I don't know if this even need `mint` and `burn` functions.

Reply:
`burn` and `mint` functions (and `burnFrom`) impact the total supply. Which is important to keep history of for specific purposes. 

Examples include:
Voting Power
Dividend Share

Which both are dependent on the ratio of held tokens at a specified time. In which knowing the total supply is necessary.